changelog/18752.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+**Redis ElastiCache DB Engine**: Renamed configuration parameters for disambiguation; old parameters still supported for compatibility.
```

Comment:
@maxcoulombe - with this formatting, this should be a "feature" not "improvement". Could you please update this file so that it's generated correctly for the 1.13 release? Thanks!